Source/FieldSolver/FiniteDifferenceSolver/EvolveB.cpp

```diff
@@ -252,25 +252,31 @@ void FiniteDifferenceSolver::EvolveRhoCartesianECT (
             [=] AMREX_GPU_DEVICE(int i, int j, int k) {
```

Comment:
At the beginning of `FiniteDifferenceSolver::EvolveRhoCartesianECT`:
Let's also add a check here that aborts if this is not a 2D or 3D simulation.